controller-test.js

```diff
@@ -0,0 +1,33 @@
+//const mongoose = require('mongoose');
```

Comment:
Good ideas here. This feature should be tested, however, could you refactor these tests to be unit tests using [Jest](https://jestjs.io/)?